libs/api/domains/driving-license/src/lib/graphql/newDrivingLicenseResult.model.ts

```diff
@@ -0,0 +1,14 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+import { IsString, IsOptional, IsBoolean } from 'class-validator'
+
+@ObjectType()
+export class NewDrivingLicenseResult {
+  @Field()
+  @IsBoolean()
+  success!: boolean
+
+  @Field({ nullable: true })
+  @IsOptional()
+  @IsString()
+  errorMessage!: string | null
+}
```

Comment:
Are the class validators on the return type possibly a mistake? Or am I missing something?